satellite/satellitedb/repairqueue.go

```diff
@@ -35,13 +34,12 @@ func (r *repairQueue) Insert(ctx context.Context, seg *pb.InjuredSegment) error
 }
 
 func (r *repairQueue) postgresSelect(ctx context.Context) (seg *pb.InjuredSegment, err error) {
-	err = r.db.QueryRowContext(ctx, r.db.Rebind(`
-	UPDATE injuredsegments SET attempted = ? WHERE path = (
+	err = r.db.QueryRowContext(ctx, `
+	UPDATE injuredsegments SET attempted = now() WHERE path = (
```

Comment:
This will be a subtle problem: because `attempted` is a timestamp without time zone, it will get a different value depending on what the db server considers to be the local time zone.

If it's an option to change to `TIMESTAMP WITH TIME ZONE`, that is to be preferred-- it won't matter what the server or your machine thinks is the local time zone, since any local times will have the time zone info annotated, and the db can always find the "real" time from that.

But since that's probably not an option, I suggest you use `now() AT TIME ZONE 'utc'` instead of `now()` (both on this line and down on line 40)

Reply:
Bill and me tried a few different statements on the database. We should use `timezone('utc', now())`

```
SELECT attempted, timezone('utc', now()) FROM injuredsegments WHERE attempted IS NOT NULL AND attempted > timezone('utc', now()) - interval '1 hour';
         attempted          |          timezone
----------------------------+----------------------------
 2019-05-21 16:08:30.609984 | 2019-05-21 16:09:43.998147
```

Reply:
updated to timezone('utc', now())

Reply:
That should be equivalent. But does that mean `now() AT TIME ZONE 'utc'` does not work? That would suggest a larger problem.

Reply:
timezone('utc', now()) works.  jen's test is successful when it returns a result.

Reply:
Right, but does `now() AT TIME ZONE 'utc'` _not_ work? That's the more standard (and also easier to read imo) way to write it

Reply:
`now() AT TIME ZONE 'utc' works as well. Only `now()` is not working.

Reply:
Ok. Just making sure.

Reply:
oh, they should be equivalent.  I was just going the opposite direction on 'easier to read'

Reply:
To my feeble mind, "`timezone(foo)`" always seems like it's going to return a time zone. But it's totally a matter of preference and what you've got should work fine.